packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -209,6 +212,14 @@ backbone = keras_nlp.models.Backbone.from_preset("hf://${model.id}")
 `,
 ];
 
+export const tf_keras = (model: ModelData): string[] => [
+	`# Note: keras<3.x must be installed
```

Comment:
Maybe clarify with an additional comment that this corresponds to (insert github link) keras 2 stuff

Reply:
yes and 'keras=2 or keras=1 must be installed' might be clearer actually

Reply:
Clarified in https://github.com/huggingface/huggingface.js/pull/774/commits/63f6f0cda07e82e4974b004bdf68807599ce89c7